packages/block-library/src/site-title/edit.js

```diff
@@ -9,14 +11,15 @@ import { Button } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
 import { RichText } from '@wordpress/block-editor';
 
-export default function SiteTitleEdit() {
+export function SiteTitleEdit( props ) {
```

Comment:
We should follow the style of the other components in this package and destructure `props`.